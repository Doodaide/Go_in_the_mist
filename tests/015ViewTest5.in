who
place A19
view
place S18
view
place R1
view
place B2
view
place C10
view
place J17
view
place Q11
view
place K3
view
term