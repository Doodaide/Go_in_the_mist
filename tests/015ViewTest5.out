B
M16,.................................................
E18,xxxxxxxxxxxxxx...................................
G3,..........................................xxxxxxx
L17,xxxxxxx..........................................
B18,xxxxxxxxxxxxxxxx#....xx.....xx.....xx.....xx.....
C9,x......x......x..#...x......x......x......x......
S9,....xxx.#..xxx....xxx....xxx....xxx....xxx....xxx
P1,..........................#.xxxxxxxxxxxxxxxxxxxxx
